dace/config_schema.yml

```diff
@@ -461,7 +461,10 @@ required:
                         title: Target frequency for Xilinx kernels
                         description: >
                             Target frequency when compiling kernels for Xilinx.
-                            Will not necessarily be achieved in practice.
+                            Will not necessarily be achieved in practice. To
```

Comment:
Can you also add that frequencies must be expressed in MHz?

Reply:
This has been added!